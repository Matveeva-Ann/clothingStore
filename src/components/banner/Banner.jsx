import Slide1 from './Slide1/Slide1';

export default function Banner() {
  return (
    <Slide1></Slide1>
  )
}