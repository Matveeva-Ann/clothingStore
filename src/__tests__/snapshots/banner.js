import React from 'react';
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer'; // Додайте імпорт renderer
import Banner from '../../components/banner/Banner.jsx';

test('Banner renders correctly', () => {
  render(
    <BrowserRouter>
      <Banner />
    </BrowserRouter>
  );

  expect(screen.getByTestId('slide1')).toBeInTheDocument();

  const tree = renderer.create(
    <BrowserRouter>
      <Banner />
    </BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
