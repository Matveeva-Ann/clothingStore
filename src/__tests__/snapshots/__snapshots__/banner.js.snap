// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<div
  className="css-e0dlsq"
  data-testid="slide1"
>
  <div
    className="css-rtya7d"
  >
    <p
      className="css-19d397d"
    >
      T-shirt / Tops
    </p>
    <h2
      className="css-2fc4s7"
    >
      Summer Value Pack
    </h2>
    <p>
      cool / colorful / comfy
    </p>
    <a
      className="css-1j6zs67"
      color="white"
      colortext="black"
      href="/Shop"
      onClick={[Function]}
      size="normal"
    >
      Shop Now
    </a>
  </div>
</div>
`;
